File: spring-certification/src/main/java/com/stackroute/springbootcertification/domain/Certificate.java
package com.stackroute.springbootcertification.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Certificate {
    private String certificateName;
    private String certificateAuthority;
    private String licenseNumber;
    private String fromDate;
    private String toDate;
}

File: Angular/src/app/add-module/certificate-dialog/domain/certificate.ts
export class Certificate {
    constructor(private certificateName: string,
        private certificateAuthority: string,
        private licenseNumber: string,
        private fromDate: string,
        private toDate: string) {
        this.certificateName = certificateName;
        this.certificateAuthority = certificateAuthority;
        this.licenseNumber = licenseNumber;
        this.fromDate = fromDate;
        this.toDate = toDate;
    }
}
